tests/example/jsonfile/jsonout.dl

```diff
@@ -0,0 +1,16 @@
+.type List = [
+    head : number,
+    tail : List
+]
+.decl A(x : List)
+A(nil).
+A([1,nil]).
+A([2,[3,nil]]).
+.output A(IO=jsonfile,filename="A")
```

Comment:
Is 'filename' being used? If it is, I would expect the output file to be `A`, not the more helpful `A.json`.